### PR TITLE
Feature/perfomance audit

### PR DESCRIPTION
**Pull request by blessedjasonmwanza**
## Feature/perfomance audit
In this update, @NataliaPoletaeva  and I (@blessedjasonmwanza ) collaborated to carry out database performance audits. Below are activities performed following [task guidelines](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_performance_audit.md) ;

- Run queries to check performance response for the following data points : 
   - `SELECT COUNT(*) FROM visits where animal_id = 4;`
   - `SELECT * FROM visits where vet_id = 2;`
   - `SELECT * FROM owners where email = 'owner_18327@mail.com';`
 
- Take screenshots for each query showing results as illustrated here-under:
  - `SELECT COUNT(*) FROM visits where animal_id = 4;`
  1st result
  _After optimization_
  
  
  - `SELECT * FROM visits where vet_id = 2;`
  2nd result
_After optimization_
  
  
  - `SELECT * FROM owners where email = 'owner_18327@mail.com';`
  3rd result
_After optimization_
  3rd result

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -71,4 +71,7 @@ CREATE TABLE visits (
     date_visited DATE
 );
```
Comment (SajjadAhmad14) [inline schema.sql:72]: Please add two screenshots for each query so there should be a total of six screenshots.
The first screenshot should show how much time the query takes BEFORE adding the indexes for foreign keys.
The second screenshot should show how much time the query takes AFTER the improvements introduced. 
The execution time in the first screenshot should not be lower than 1 sec (1000 ms).
The execution time in the second screenshot should be significantly lower than 1 sec (1000 ms).

File: schema.sql [hunk 2]
```diff
@@ -71,4 +71,7 @@ CREATE TABLE visits (
...
 ALTER TABLE visits ADD CONSTRAINT animals FOREIGN KEY (animal_id) REFERENCES animals(id);
-ALTER TABLE visits ADD CONSTRAINT vet FOREIGN KEY (vet_id) REFERENCES vets(id);
\ No newline at end of file
+ALTER TABLE visits ADD CONSTRAINT vet FOREIGN KEY (vet_id) REFERENCES vets(id);
+
```
Comment (SajjadAhmad14) [inline schema.sql:75]: Please add indexes to foreign keys as this is the basic solution to increase the performance of a table.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi team, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
PR has a title and description. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: schema.sql (hunk 1 above)
Comment (blessedjasonmwanza) [reply schema.sql:72]: Thanks a lot @SajjadAhmad14 😃  Working on it right now />

File: schema.sql (hunk 2 above)
Comment (blessedjasonmwanza) [reply schema.sql:75]: Actioned.

File: schema.sql (hunk 1 above)
Comment (blessedjasonmwanza) [reply schema.sql:72]: <Actioned/>

**Review by rukundoeric — APPROVED** [review]
Hi @blessedjasonmwanza,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------
>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**